src/plugins/workspace/public/components/workspace_creator/workspace_permission_setting_panel.tsx

```diff
@@ -263,8 +264,12 @@ export const WorkspacePermissionSettingPanel = ({
     return result;
   }, [value]);
 
+  // default permission mode id admin
```

Comment:
Is this a requirement? The permission mode set to `Admin` by default is a little bit strange.

Reply:
The EuiButtonGroup needs a default value, is it okay to assume that `Admin` is the default permission mode?

Reply:
Changed default permission mode to `Read`